templates/pages/setup/notification/mailing_setting.html.twig

```diff
@@ -356,19 +356,72 @@
                 <i class="ti ti-device-floppy"></i>
                 <span>{{ _x('button', 'Save') }}</span>
             </button>
-            <button class="btn btn-outline-secondary me-2" type="submit" name="test_smtp_send" value="1">
+            <button type="button" id="test-email-btn" class="btn btn-outline-secondary me-2" name="test_smtp_send" value="1">
                 {{ __('Send a test email to the administrator') }}
             </button>
-        </div>
-    </form>
-{% else %}
-    <div class="row">
-        <div class="col">
-            <div class="alert alert-info">
-                <i class="fas fa-info-circle fa-2x alert-icon"></i>
-                {{ __('Notifications are disabled.') }}
-                <a href="{{ config('root_doc') }}/front/setup.notification.php">{{ __('See configuration') }}</a>
+
+            <!-- Modal -->
+            <div class="modal fade" id="mailTestLogModal" tabindex="-1" role="dialog" aria-labelledby="mailTestLogModalLabel" aria-hidden="true">
+                <div class="modal-dialog" role="document">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <h5 class="modal-title" id="mailTestLogModalLabel">{{ __('Mail Test Logs') }}</h5>
+                            <button type="button" class="btn-close" data-bs-dismiss="modal"></button>
+                        </div>
+                        <div class="modal-body">
+                            <div>
+                                <h4>Result</h4>
+                                <span class="badge mb-3">{{ __('Success') }}</span>
+                            </div>
+                            <div>
+                                <h4>Error</h4>
+                                <pre style="white-space: pre-wrap;"></pre>
+                            </div>
+                            <div>
+                                <h4>Logs</h4>
+                                <pre></pre>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <script>
+                    $('#test-email-btn').on('click', function () {
+                        const $modal = $('#mailTestLogModal');
+
+                        $.ajax({
+                            url: CFG_GLPI.root_doc + '/ajax/notificationmailingsettings.php',
+                            method: 'POST',
+                            datatype: 'json',
```

Comment:
This is useless as long as the `Content-Type` header send in response is `application/json`.
```suggestion
```